Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: build
+build:
+	mkdir -p bin && export CGO_ENABLED=0 GOOS=linux && go build -o bin/app -ldflags '-s -w' ./core/main.go
```

Comment:
Could you add more about building job and swagger, cleaning and making images?

Reply:
added